### PR TITLE
ENH keep intel openmp so

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

File: recipe/build-llvm-openmp.sh [hunk 1]
```diff
@@ -22,6 +22,9 @@ make install
 rm -f $PREFIX/lib/libgomp$SHLIB_EXT
 rm -f $PREFIX/lib/libiomp5$SHLIB_EXT
 
+# make a symlink
+ln -s $PREFIX/lib/libomp$SHLIB_EXT $PREFIX/lib/libiomp5$SHLIB_EXT
```
Comment (isuruf) [inline recipe/build-llvm-openmp.sh]: I think this should go in https://github.com/conda-forge/_openmp_mutex-feedstock

Comment (beckermr) [reply recipe/build-llvm-openmp.sh]: We need it on osx and linux, so should we create the mutex for osx too?

Comment (isuruf) [reply recipe/build-llvm-openmp.sh]: Ah, that's okay then. I would just remove line 23.

**Comment by beckermr** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
Can you do the same for windows?

**Comment by beckermr** [comment]
Check out the latest commit. I know nothing about windows, so it may very well be very wrong. :/

**Comment by isuruf** [comment]
Thanks

**Pull request merged**